nautobot_data_validation_engine/models.py

```diff
@@ -366,3 +367,65 @@ def clean(self):
 
         if getattr(model_field, "unique", False):
             raise ValidationError({"field": "This field is already unique by default."})
+
+
+class AuditRule(PrimaryModel):
+    """Model to represent the results of a audit method."""
+
+    class_name = models.CharField(max_length=100, blank=False, null=False)
+    method_name = models.CharField(max_length=100, blank=False, null=False)
+    last_validation_date = models.DateTimeField(blank=False, null=False)
+    content_type = models.ForeignKey(ContentType, on_delete=models.CASCADE, blank=False, null=False)
```

Comment:
Standard for these use cases is `on_delete=models.PROTECT`. Consider the case where a plugin is removed from any environment and the content type becomes eligible for purging. Once it is, referenced `AuditRule`s would be also deleted. Given the audit context, we likely don't want that.